test/torch_np/numpy_tests/core/test_multiarray.py

```diff
@@ -3613,8 +3613,9 @@ def test_ip_types(self):
     def test_mask_size(self):
         assert_raises(ValueError, np.putmask, np.array([1, 2, 3]), [True], 5)
 
-    @parametrize("dtype", (">i4", "<i4"))
-    def test_byteorder(self, dtype):
+    @parametrize("dtype", (True, False))
+    def test_byteorder(self, greater):
+        dtype = ">i4" if greater else "<i4"
```

Comment:
Could you also fix/rewrite scripts/compile_tests/update_failures.py to work with the new format?